Mantle/MTLModel+NSCoding.m

```diff
@@ -37,7 +37,11 @@ static BOOL coderRequiresSecureCoding(NSCoder *coder) {
 // Returns all of the given class' encodable property keys (those that will not
 // be excluded from archives).
 static NSSet *encodablePropertyKeysForClass(Class modelClass) {
+#ifdef GNUSTEP
+    return [[modelClass encodingBehaviorsByPropertyKey] keysOfEntriesPassingTest:(GSKeysAndObjectsPredicateBlock) ^BOOL(NSString *propertyKey, NSNumber *behavior, BOOL *stop) {
```

Comment:
Mismatched indentation

Reply:
Fixed by replacing all instances of `#if GNUSTEP` where we cast the enumeration blocks by using explicitly typed `NSDictionary<T1, T2>` variables, instead